### PR TITLE
[JENKINS-36169] Override resultItem css to remove border and padding

### PR DESCRIPTION
**Pull request by scherler**
**Decription**
- [x] Line numbers should appear in gutter
- [x] Remove the random border, fix the padding so it doesn't look disjointed

https://issues.jenkins-ci.org/browse/JENKINS-36169

screenshot from 2016-07-25 14-33-46

**Submitter checklist**
- [x] Change is code complete and matches issue description
- [x] Apppropriate unit or acceptance tests or explaination to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

**Reviewer checklist**
- [x] Run the changes and verified the change matches the issue description
- [x] Reviewed the code
- [x] Verified that the appropriate tests have been written or valid explaination given

@reviewbybees

### CODE REVIEW

**Comment by i386** [comment]
@scherler could we remove the gutter from the default result item? It was there for line numbers in the log but is not needed in the base component if we are overriding it for line numbers.

**Comment by i386** [comment]
@scherler the gutter is missing its own background colour (#4A4A4A) as per the mockup 
group

**Comment by michaelneale** [comment]
🐝  from me as this looks good. 

There is a little more space than needs be in freestyle logs 
screen shot 2016-07-26 at 2 33 48 pm

but it doesn't *feel* unbalanced to me.

**Comment by i386** [comment]
Please address/refute my previous comment before I give a 🐝

**Comment by scherler** [comment]
@i386: 
screenshot from 2016-07-26 10-42-20

The remove of the style and the change to make the screenshot possible will be addressed in https://github.com/jenkinsci/jenkins-design-language/pull/81 since the corresponding changes are there.

**Comment by scherler** [comment]
@michaelneale regarding the spacing that is as well discussed in jdl

**Comment by michaelneale** [comment]
OK I think addressed in JDL comments 🐝

File: blueocean-dashboard/src/main/js/components/Step.jsx [hunk 1]
```diff
@@ -122,7 +122,11 @@ export default class Node extends Component {
                   logArray={log.logArray}
                   scrollToBottom={scrollToBottom}
                   prefix={`step-${id}-`}
-                /> } &nbsp;
+                /> }
+
+                { !log && <span>
+                    &nbsp;
```
Comment (kzantow) [inline blueocean-dashboard/src/main/js/components/Step.jsx]: :'(

**Comment by kzantow** [comment]
LGTM, too :bee:

**Comment by i386** [comment]
Looks good

**Comment by scherler** [comment]
@reviewbybees done

**Pull request merged**